gnd/src/main/res/layout/date_input_field.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright 2021 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <data>
+    <variable
+      name="viewModel"
+      type="com.google.android.gnd.ui.editobservation.DateFieldViewModel" />
+  </data>
+
+  <LinearLayout
+    style="@style/FieldView"
+    android:orientation="vertical">
+
+    <com.google.android.material.textfield.TextInputLayout
+      android:id="@+id/text_input_layout"
+      style="@style/EditObservation.Field.Text"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:hint="@{viewModel.fieldLabel()}">
+
+      <com.google.android.material.textfield.TextInputEditText
+        android:id="@+id/text_input_edit_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:maxLines="1"
+        android:enabled="true"
+        android:inputType="datetime"
+        android:clickable="true"
+        android:focusableInTouchMode="false"
+        android:focusable="false"
+        android:text="@{viewModel.responseText}"
+        app:error="@{viewModel.error}"
+        tools:text="Date Time" />
```

Comment:
Please move text label to string resources instead.

Reply:
moved to strings.xml